packages/amplication-client/src/Resource/constants.ts

```diff
@@ -139,7 +139,10 @@ export function prepareServiceObject(
           generateRestApi: generateRestApi,
           serverPath: serverDir,
         },
-        authProvider: EnumAuthProviderType.Jwt,
+        authProvider:
+          auth === "core"
+            ? EnumAuthProviderType.Jwt
+            : EnumAuthProviderType.Http,
```

Comment:
@morhag90 can you look at this please?

Reply:
@abrl91 Looks fine for now (in the future, if we add another option besides the "core," we will need to update this logic)